interpreter/src/main/java/org/qbicc/interpreter/impl/VmImpl.java

```diff
@@ -185,6 +186,7 @@ public final class VmImpl implements Vm {
         ClassContext bcc = ctxt.getBootstrapClassContext();
         classClass = new VmClassClassImpl(this);
         objectClass = classClass.getSuperClass();
+        reflectionDataClass = new VmClassImpl(this, classClass, bcc.findDefinedType("java/lang/Class$ReflectionData").load());
```

Comment:
This should have a `postConstruct` call down below (I think). Also the two-arg ctor should be OK since the `classClass` field is initialized at this point.

Reply:
Good catch on the postConstruct.  Fixed.